contracts/Lens/RewardLens.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.5.16;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @notice This is a helper for fetching and maintaing supply/borrow rewards for lending markets.
+ */
+contract RewardLens is Ownable {
+    mapping(address => uint256) public supplyRewardsJoePerSec;
+    mapping(address => uint256) public borrowRewardsJoePerSec;
+    mapping(address => uint256) public supplyRewardsAvaxPerSec;
+    mapping(address => uint256) public borrowRewardsAvaxPerSec;
```

Comment:
not sure what the performance implications are but another way to store this would be a `mapping(address => SomeRewardsStruct)`. wondering if anyone knows which one is better performance wise since I think the struct mapping is a bit cleaner code-wise. curious to hear what folks think

Reply:
Oh right I think this does make more sense! Tried to read up on it and seems like this is more gas efficient 👌